flowey/flowey_hvlite/src/pipelines_shared/gh_flowey_bootstrap_template.yml

```diff
@@ -19,7 +19,7 @@
 - run: |
     set -x
     i=0; while [ $i -lt 5 ] && ! sudo apt-get update; do let "i=i+1"; sleep 1; done;
-    sudo apt-get -o DPkg::Lock::Timeout=60 install gcc -y
+    sudo apt-get -o DPkg::Lock::Timeout=60 install build-essential -y
```

Comment:
do not work around downstream dependency issues by updating the bootstrap template. if a specific flowey node needs something from build-essential (presumably, a linker or something?), it should explicitly claim it

Reply:
I was just changing this for consistency since installing gcc is basically the same as installing build-essential

Reply:
that can't be right? I'm sure build-essential installs a lot more than just gcc?

Reply:
Ok I will revert this - there are a few more (just tested on a fresh ubuntu install, I couldn't figure out how to recursively list apt dependencies without having the packages not installed). I can't seem to find any documentation that says exactly which packages are needed for rust to work, but I suppose "gcc" plus all of the other stuff it pulls in is good enough for now.